src/sparseml/pytorch/optim/modifier_distillation.py

```diff
@@ -253,12 +261,15 @@ def loss_update(
             (calculate batch number using this and epoch)
         :return: loss tensor with knowledge distillation loss added
         """
-        loss = super().loss_update(
+        self._student_loss = None
```

Comment:
since these are for logging purposes, can we rename to something like `latest_*`